src/Microsoft.DotNet.XHarness.CLI/Commands/Apple/AppleInstallCommand.cs

```diff
@@ -0,0 +1,68 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.DotNet.XHarness.Apple;
+using Microsoft.DotNet.XHarness.CLI.CommandArguments.Apple;
+using Microsoft.DotNet.XHarness.Common.CLI;
+using Microsoft.DotNet.XHarness.Common.Logging;
+using Microsoft.DotNet.XHarness.iOS.Shared;
+using Microsoft.DotNet.XHarness.iOS.Shared.Execution;
+using Microsoft.DotNet.XHarness.iOS.Shared.Logging;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.DotNet.XHarness.CLI.Commands.Apple
+{
+    internal class AppleInstallCommand : AppleAppCommand<AppleInstallCommandArguments>
+    {
+        private const string CommandHelp = "Installs a given iOS/tvOS/watchOS/MacCatalyst application bundle in a target device/simulator";
+
+        protected override AppleInstallCommandArguments AppleAppArguments { get; } = new();
+
+        protected override string CommandUsage { get; } = "apple install [OPTIONS]";
```

Comment:
If I remember correctly we agreed to explicitly put here mandatory parameters. Could you please add them in some upcoming PR?